Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -15,7 +15,7 @@
 
 ''' GLOBALS '''
 
-BASE_URL = 'http://api.domaintools.com'
+BASE_URL = demisto.params().get('base_url', 'http://api.domaintools.com')  # we keep old http url for backwards comp
```

Comment:
i think the default value here will never be taken since `base_url` will exist in the params dict, just with empty string
if we want default value we should set it as : `demisto.params().get('base_url') or 'http://api.domaintools.com'`

Reply:
Not exactly. See: https://github.com/demisto/etc/issues/30318. But i will change to future proof.